internal/pkg/v2/infrastructure/redis/event.go

```diff
@@ -20,6 +20,11 @@ const EventsCollection = "v2:event"
 
 // ************************** DB HELPER FUNCTIONS ***************************
 func addEvent(conn redis.Conn, e model.Event) (addedEvent model.Event, edgeXerr errors.EdgeX) {
+	_, edgeXerr = eventByID(conn, e.Id)
```

Comment:
Please add comment `why` doing this. Then the code make sense quickly to reader.

Reply:
added comment